packages/rancher-monitoring/package.yaml

```diff
@@ -1,2 +1,3 @@
 url: https://kubernetes-charts.storage.googleapis.com/prometheus-operator-8.16.1.tgz
 packageVersion: 00
+splitCRDsIntoSeparatePackage: true
```

Comment:
nit: somewhat verbose, maybe createCRDChart or something, but its fine

Reply:
Should be a quick change, will update in the next commit!

Reply:
This looks pretty cool. I like `generateCRDChart.enabled`, but either works.

Reply:
I like that better too. Updated in the latest commit

Reply:
IMO this doesn't need to dot-splited words. Feel like the single word is enough since we are not adopting the helm value style for this package.yaml.